docs/user/learner.rst

```diff
@@ -41,27 +44,43 @@ Learn
 
 Each time you login into **Kolibri**, the first thing you will see is the **Learn** page. Here you will find learning topics and materials related to what you were doing the last time you used Kolibri, or those recommended by your teachers and coaches.
 
+The content you see in the **Learn** page will be different for each channel. Change the channel to explore the resto of content you can use in Kolibri.
+
 .. image:: img/learn.png
 	:alt: learn page
 
-Explore
-~~~~~~~~
+Recommended
+***********
 
-**Explore** page offers you the option to navigate through the complete set of learning topics and materials available in **Kolibri**. Use it as you wish, or according to indications from your teachers and coaches. 
+In the **Recommended** tab you can see two sections:
 
-.. image:: img/explore.png
-	:alt: explore page
+* **All content** section where you can browse through all learning materials in the currently selected channel by using the **Previous** and **Next** links.
+* **Most popular** section which displays the most frequently used materials in your school or facility.
 
-Navigate Kolibri content with breadcrumb links
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Topics
+****** 
 
-**Breadcrumb** links indicate which group of **Kolibri** content you are viewing in each moment. 
+**Topics** tab offers you the option to navigate through the complete set of learning topics and materials available in Kolibri, for the currently selected channel. Use it as you wish, or according to indications from your teachers and coaches. 
 
-To go back (up one level from where you are currently), click the next to last link in the breadcrumb. 
+.. image:: img/explore.png
+	:alt: topics tab
+
+Navigate Kolibri topics with breadcrumb links
+*********************************************
+
+When you are browsing a topic in Kolibri, the **Breadcrumb** links indicate all its parent topics. To go back (up one level from where you are currently), click the next to last link in the breadcrumb. 
```

Comment:
Good point about the "up" metaphor! :+1: 
`every topic visited prior` sounded a bit too strong, let me know if this latest edit sounds clearer.